src/components/IntroPopup/IntroPopup.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react'
+import { Button, Column, Header, Popup } from 'decentraland-ui'
+import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Props, State } from './IntroPopup.types'
+import './IntroPopup.css'
+
+const POLYGON_INTRO_POPUP_LOCAL_STORAGE_KEY = 'polygon-intro-popup'
+
+export default class IntroPopup extends PureComponent<Props, State> {
+  state: State = {
+    open: !localStorage.getItem(POLYGON_INTRO_POPUP_LOCAL_STORAGE_KEY)
+  }
+
+  handleDismiss = () => {
+    localStorage.setItem(POLYGON_INTRO_POPUP_LOCAL_STORAGE_KEY, '1')
+    this.setState({ open: false })
+  }
+
+  render() {
+    const { open } = this.state
+    return (
+      <>
+        {open ? <div className="IntroPopupOverlay" /> : null}
+        <Popup
+          position="top center"
+          open={open}
+          className="IntroPopup"
+          content={
+            <Column>
+              <Header>{t('intro_popup.title')}</Header>
+              <p className="content">
+                {
+                  <T
+                    id="intro_popup.content"
+                    values={{
+                      br: <br />,
+                      blog_link: (
+                        <a href="https://decentraland.org/blog/announcements/polygon-mana/" target="_blank" rel="no:opener no:referrer">
+                          {t('intro_popup.blog_link')}
+                        </a>
+                      ),
+                      account_link: (
+                        <a href="https://account.decentraland.org" target="_blank" rel="no:opener no:referrer">
```

Comment:
I think this is fine, as this popup (supposedly) will not be here forever, but account.decentralad.org already exists as an env variable in REACT_APP_ACCOUNT_URL